Classes/Fluid/ViewHelper/ArgumentDefinition.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace SMS\FluidComponents\Fluid\ViewHelper;
+
+class ArgumentDefinition extends \TYPO3Fluid\Fluid\Core\ViewHelper\ArgumentDefinition
+{
+    /**
+     * JSON schema definition
+     *
+     * @var string
+     */
+    protected $schema;
```

Comment:
Technically, if you don't assign a value here (like ''), it is not a string but a mixed. I would prefer here an assignment of = '' and later on in the has-function I would use !empty().